### ISSUE TITLE
Web-mode uses 1 space indent instead of 2

### ISSUE THREAD

**Issue by krzysztofzuraw** [fxbois/web-mode#848]
Hey,
I have this config in my `init.el`:
```
(setq-default indent-tabs-mode nil)
(add-to-list 'auto-mode-alist '("\\.html?\\'" . web-mode))
(defun my-web-mode-hook ()
  "Hooks for Web mode."
  (setq web-mode-markup-indent-offset 2)
  (setq web-mode-css-indent-offset 2)
  (setq web-mode-code-indent-offset 4)
  (setq web-mode-enable-auto-pairing t)
  (setq web-mode-enable-auto-closing t)
  (setq web-mode-enable-current-element-highlight t)
  (setq web-mode-enable-current-column-highlight t)
)
(add-hook 'web-mode-hook  'my-web-mode-hook)
```
And when I have `<script>` tag I only have javascript code indented by 1 space instead of 2. Is there a way to fix that?

**Comment by fxbois** [comment]
@krzysztofzuraw please ask on stackoverflow

**Comment by builtinnya** [comment]
For someone getting into this issue:

- **"Left padding"** section on [the official documentation](http://web-mode.org/)
- [How can I indent inline Javascript in web-mode? - Stack Overflow](https://stackoverflow.com/questions/36701024/how-can-i-indent-inline-javascript-in-web-mode)

**TL;DR**

```elisp
(add-hook 'web-mode-hook
  (lambda ()
    (setq web-mode-script-padding 2
          web-mode-style-padding 2)))
```
